assets/styles/label-landscape.scss

```diff
@@ -38,15 +38,15 @@ th {
 
 .prices {
   position: relative;
-  margin-bottom: 0.5rem;
   padding-bottom: 1rem;
   border-top: 1px black solid;
   border-bottom: 1px black solid;
+  margin-bottom: 0.5rem;
 }
 
 .prices:after {
-  content: " ";
   display: table;
+  content: " ";
```

Comment:
Please either use single or double quotes. (https://github.com/netzstrategen/woocommerce-price-labels/pull/12/files#diff-87b4c02c3ad5efaa75c8e298c5519007R173). I think our coding standard is single ones.